packages/flutter/lib/src/material/material.dart

```diff
@@ -517,7 +521,7 @@ class _RenderInkFeatures extends RenderProxyBox implements MaterialInkController
   }
 
   @override
-  bool hitTestSelf(Offset position) => true;
+  bool hitTestSelf(Offset position) => absorbHitTest;
```

Comment:
@HansMuller the only difference from the previous pr is that the Material.transparency will no longer absorb hittest.